webapp/security/helpers.py

```diff
@@ -1,13 +1,19 @@
+# Standard library
```

Comment:
Everything in here has already been reviewed, I've only moved the existing functions from views.py to this file to make the `cve_index` view dryer and more readable